### PR TITLE
package-readme: prevent notice synopsis

### PR DESCRIPTION
**Pull request by diggy**
e.g.

```
PHP Notice:  Undefined index: synopsis in /Users/chewbacca/.wp-cli/packages/vendor/wp-cli/scaffold-package-command/inc/ScaffoldPackageCommand.php on line 277
```

### CODE REVIEW

File: inc/ScaffoldPackageCommand.php [hunk 1]
```diff
@@ -273,7 +273,7 @@ public function package_readme( $args, $assoc_args ) {
 				$readme_args['commands'][] = array(
 					'name' => "wp {$command}",
 					'shortdesc' => $parent_command['description'],
-					'synopsis' => "wp {$command} {$parent_command['synopsis']}",
+					'synopsis' => "wp {$command}" . ( empty( $parent_command['subcommands'] ) ? " {$parent_command['synopsis']}" : "" ),
```
Comment (danielbachhuber) [inline inc/ScaffoldPackageCommand.php]: Shouldn't this be `! empty( $parent_command['synopsis'] )`?

Comment (diggy) [inline inc/ScaffoldPackageCommand.php]: @danielbachhuber cf. https://github.com/wp-cli/wp-cli/blob/master/php/commands/cli.php#L42

Comment (danielbachhuber) [inline inc/ScaffoldPackageCommand.php]: Ok. Can you add a test case for this change?

Comment (diggy) [inline inc/ScaffoldPackageCommand.php]: sure, will do asap

Comment (diggy) [inline inc/ScaffoldPackageCommand.php]: @danielbachhuber Actually, I'm not exactly sure how to test this, or if it should be tested at all. The difference lies in commands running in the namespace, and subcommands. To test this you'd need two commands e.g. `brew` and `brew this`. This "bug" results in a space after the namespace command synopsis, like [here](https://github.com/wp-cli/admin-command#using).

Comment (danielbachhuber) [inline inc/ScaffoldPackageCommand.php]: > Actually, I'm not exactly sure how to test this, or if it should be tested at all.

Ok, that's reasonable. If it's a small change, I'm typically fine not including tests if it's too hard to test for the change.

**Pull request merged**